old-tutorials-in-development/tutorials-in-development/R/dc-spatio-temporal-intro/05-geographic-vs-projected-CRS.Rmd

```diff
@@ -169,22 +189,22 @@ Spatial projection refers to the mathematical calculations
 performed to flatten the 3D data onto a 2D plane (our computer screen
 or a paper map). Projecting data from a round surface onto a flat surface, results
 in visual modifications to the data when plotted on a map. Some areas are stretched
-and some some are compressed. We can see this when we look at a MAP of the entire
+and some some are compressed. We can see this when we look at a map of the entire
 globe.
 
 
 The mathematical calculations used in spatial projections are designed to
 optimize the relative size and shape of a particular region on the globe.
 
 <figure>
-    <a href="http://www.progonos.com/furuti/MapProj/Normal/CartDef/MapDef/Img/devSurfaces.png">
-    <img src="http://www.progonos.com/furuti/MapProj/Normal/CartDef/MapDef/Img/devSurfaces.png">
+    <a href="https://www.earthdatascience.org/images/courses/earth-analytics/spatial-data/spatial-projection-transformations-crs.png">
+    <img src="https://www.earthdatascience.org/images/courses/earth-analytics/spatial-data/spatial-projection-transformations-crs.png" alt="A 3-dimensional globe transformed into a cone creates a half-circle map while a cylinder creates a rectangular map.">
     </a>
     <figcaption>The 3-dimensional globe must be transformed to create a flat
     2-dimensional map. How that transformation or **projection** occurs changes
     the appearance of the final map and the relative size of objects in
     different parts of the map.  
-    Source: CA Furuti, progonos.com/furuti</figcaption>
+    Source: C.A. Furuti, website formerly at progonos.com/furuti</figcaption>
```

Comment:
@donal-at-NEON: The source website no longer exists but the map should still be credited to Furuti.  I wasn't sure how to handle this so I copied [what NASA did](https://www.giss.nasa.gov/tools/gprojector/help/credits.html) to credit him: 
"Furuti, C.A.: Map Projections. Website formerly at "www.progonos.com/furuti/MapProj/". Last accessed 2018-07-19. No longer available."

Reply:
That works for me, thanks for finding a good solution here.